meta/runtime.yml

```diff
@@ -132,3 +133,188 @@ plugin_routing:
       deprecation:
         removal_version: 2.0.0
         warning_text: see plugin documentation for details
+    onyx_aaa:
+      deprecation:
```

Comment:
No need for the `deprecation:` section, this should be the same as https://github.com/ansible/ansible/pull/70213/files#diff-e5cc2d385786a9f022d91173b5d91469R2463

Reply:
Isn't it better to have a deprecation statement here? Otherwise we can never ever remove these redirects, since people will not know that the modules are no longer in community.network. (Or is Ansible issuing deprecation warnings when it encounters redirects in collections?)

Reply:
I think we decided on IRC that it's better to have deprecation entries. For removals later (i.e. after 1.0.0) we only want to add them some time later, but right now we add them directly.